src/exercise/server/services/ItemManager.js

```diff
@@ -0,0 +1,98 @@
+const fs = require('fs').promises;
+const PokemonClient = require('../clients/PokemonClient');
+const tasksFile = 'tasks.json';
+
+module.exports = class ItemManager {
+    constructor() {
+        this.taskId = 1;
+        this.pokemonClient = new PokemonClient();
+        this.FailedFetchPokemonError = new Error();
```

Comment:
theres no benefit in having the error instance on the class - have it outside
(or even better, in an errors folder then import them